### DISCUSSION TITLE
Feature: Select git branch

### DISCUSSION THREAD

**Discussion by franciscolourenco** [PatrickF1/fzf.fish#91] [Archive] [answered]
It would be a nice addition to be able to select git branches in addition to the git commits. ctrl+b?

Thanks!

**Comment by PatrickF1** [comment]
Noted! Thanks for the idea. I'll probably build it :)

**Comment by PatrickF1** [comment]
@franciscolourenco I'd like to ask for your opinion. Would you prefer it filter local branches only or all branches, including remote?

**Comment by franciscolourenco** [comment]
I mostly only checkout local branches, or remote branches but still locally (not like origin/branch). So I wouldn't mind remote branches to be included but only with the simple Branch name without the remote qualifier. Thanks!

**Comment by PatrickF1** [comment, marked as answer]
Hi again, I'e decided to not implement this. Reasons:
1) Fish's git autocompletion is already pretty awesome and helpful. This means the marginal utility of adding this would be small and only really helpful for people with tons of local branches.
2) I want to avoid feature cruft.

Thanks for the idea though! Please give me more ideas if you have them :)
